Source/ProcessingTest/Presenters/ProcessingPresenter.cpp

```diff
@@ -118,16 +118,19 @@ void ProcessingPresenter::Show()
     OC_LOG_INFO("Loading image");
     std::string extension = _currentFilePath.substr(_currentFilePath.find_last_of(".") + 1);
     std::transform(extension.begin(), extension.end(), extension.begin(), ::tolower);
-     
+    
+    FileFormat fileFormat;
     if(extension == "dng")
     {
-        provider->Load(_currentFilePath, FileFormat::DNG, *_image.get(), *poolAllocator);
+        fileFormat = FileFormat::DNG;    
     }
     else if(extension == "mlv")
     {
-        provider->Load(_currentFilePath, FileFormat::MLV, *_image.get(), *poolAllocator);
+        fileFormat = FileFormat::MLV;
     }
     
+    provider->Load(_currentFilePath, FileFormat::DNG, *_image.get(), *poolAllocator);

```

Comment:
**fileFormat** is not used here, also to prevent crashes assign some initial value to it